### ISSUE TITLE
cannot get internal attributes in abstraction layer mode

### ISSUE THREAD

**Issue by a23s4a** [cannatag/ldap3#437]
Hello, I cannot get internal attributes from ldap when using abstraction layer, though I can retrieve it when using classic mode:
ABSTRACTION LAYER MODE:
`r = Reader(conn, obj_person, 'ou=users,dc=domain,dc=org', query, get_operational_attributes=True, attributes=['modifyTimestamp'])`
_________________________________________________________________________________
`/usr/local/lib/python3.5/dist-packages/ldap3/abstract/cursor.py in __init__(self, connection, object_def, base, query, components_in_and, sub_tree, get_operational_attributes, attributes, controls)`
`def __init__(self, connection, object_def, base, query='', components_in_and=True, sub_tree=True, get_operational_attributes=False, attributes=None, controls=None):`
`Cursor.__init__(self, connection, object_def, get_operational_attributes, attributes, controls)`
`self._components_in_and = components_in_and`
`self.sub_tree = sub_tree`

`/usr/local/lib/python3.5/dist-packages/ldap3/abstract/cursor.py in __init__(self, connection, object_def, get_operational_attributes, attributes, controls)`
`if log_enabled(ERROR):`
`log(ERROR, '%s for <%s>', error_message, self)`
`raise LDAPCursorError(error_message)`
`self.attributes = set(attributes) if attributes else set([attr.name for attr in self.definition])`

`LDAPCursorError: Attributes 'modifyTimestamp' non in definition`

CLASSIC MODE:
`conn.search('dc=domain,dc=org', '(&(objectclass=person)(mail=somemail@domain.org))', attributes='modifyTimestamp')`
`True`
`conn.entries[0].modifyTimestamp`
`modifyTimestamp: 2017-09-25 07:14:12+00:00`

**Comment by daenney** [comment]
For me at least the request works if I change it to omit `attributes=['modifyTimestamp']` and simply do `obj_person += 'modifyTimestamp'`. The attribute is then correctly requested and I don't get the error @a23s4a gets.

However, I still can't see or get to the operation attributes in the response entry.

**Comment by cannatag** [comment]
Hi, sorry for being late with this issue. I've fixed the get_operational_attributes behaviour. It should now return all the operational attributes in an entry prefixed with the OA_ string. You can change the prefix with the ABSTRACTION_OPERATIONAL_ATTRIBUTE_PREFIX global parameter:

```
from ldap3 import set_config_parameter
set_config_parameter('ABSTRACTION_OPERATIONAL_ATTRIBUTE_PREFIX', 'MY_PREFIX')
```

**Comment by cannatag** [comment]
You must checkout the latest code in dev.

**Comment by cannatag** [comment]
Released in version 2.4.1
